internal/venice-common/src/main/resources/avro/PartitionState/v17/PartitionState.avsc

```diff
@@ -235,6 +235,11 @@
       "doc": "The last offset consumed successfully in this partition from the version topic. This is used during resubscription when the Global RT DIV feature is enabled.",
       "type": "long",
       "default": -1
+    }, {
+      "name": "heartbeatTimestamp",
+      "doc": "The most stale heartbeat timestamp among all the regions in this partition for fast online transition during server restart.",
```

Comment:
can you clarify this doc a bit ?  oldest hb timestamp ?

Reply:
Yes, this is the oldest heartbeat timestamp as I mentioned as most stale one.